src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -453,7 +453,7 @@ public List<int> ConsensusTipChanged(ChainedHeader newConsensusTip, bool blockMi
                 this.UnconsumedBlocksDataBytes -= currentHeader.Block.BlockSize.Value;
                 this.UnconsumedBlocksCount--;
 
-                this.logger.LogTrace("Size of unconsumed block data is decreased by {0}, new value is {1}.", currentHeader.Block.BlockSize.Value, this.UnconsumedBlocksDataBytes);
+                this.logger.LogTrace("UnconsumedBlocks decreased by block {0} (byte: {1}).New count: {2}; new size: {3}", currentHeader.Block.GetHash(), currentHeader.Block.BlockSize.Value, this.UnconsumedBlocksCount, this.UnconsumedBlocksDataBytes);
```

Comment:
`.New count` has a `.` (dot) did you not mean `;` or `,`

Reply:
also dont trace `header.Block.GetHash()` this will calculate hash (it might be cached yeah but still) better just trace the `ChainedHeader`  the ToString will trace the hash already

Reply:
tbh I really intended `. New count..` but it's fine to have a ,
about the hash I thought it was cached, but I prefer anyway to just print the chainedheader, so I'm updating it